.github/workflows/build-docker-typesafe.yml

```diff
@@ -0,0 +1,55 @@
+name: Fineract Docker build - Type-Safe project
```

Comment:
Let's find a proper name for this... "typesafe" is a bit too generic (mentioned it also further down)... let's come up with a name that we can use beyond the GSoC project.

Reply:
noted